cpp/src/slice2swift/Gen.cpp

```diff
@@ -241,52 +241,37 @@ Gen::TypesVisitor::visitClassDefStart(const ClassDefPtr& p)
 
     ClassDefPtr base = p->base();
 
-    out << sp;
-    out << nl << "@_documentation(visibility: internal)";
-    out << nl << "public class " << removeEscaping(name)
```

Comment:
It's not clear to me why we generated this extra class, with a single method that returns the type object.

This PR simplifies the logic by generating an extension method that returns the type object (`AnyObject.Type?`).